examples/idm/job.json

```diff
@@ -0,0 +1,25 @@
+{
+    "nevents": 10000,
+    "run_params": "2pt3",
+    "Map": 300.0,
+    "mchi": 100.0,
+    "dmchi": 10.0,
+    "seed": 879483,
+    "target_thickness": 0.002,
+    "beam_energy": 2300.0,
+    "num_electrons": 1500,
```

Comment:
The beam current used in 2016 was 200 nA which corresponds to 2500 electrons per bunch.